### PR TITLE
adding github star to footer, with some minor css

### PR DESCRIPTION
**Pull request by svwielga4**
#19 
adds star, adds some css so its not sticking out too far on the top

### CODE REVIEW

**Comment by svwielga4** [comment]
How do i make the build pass? I'm not sure what I did to break it

File: index.html [hunk 1]
```diff
@@ -25,83 +28,97 @@ <h1>WEBEXTENSIONS</h1>
...
-        </nav> 
+            <a class="github-button" href="https://github.com/web-ext-labs/ui-tool" data-icon="octicon-star" data-show-count="true" aria-label="Star ntkme/github-buttons on GitHub">Star</a>
+        </nav>
     </footer>
-    <script src="build.js" ></script>
+    <script src="build.js"></script>
 </body>
-</html>
+
+</html>
```
Comment (tsl143) [inline index.html]: We need a new line in the file.

Comment (svwielga4) [reply index.html]: added = )

**Comment by tsl143** [comment]
Thanks for PR!
Also for linting :)

**Pull request merged**